x/ccv/provider/keeper/genesis_test.go

```diff
@@ -11,7 +11,6 @@ import (
 	consumertypes "github.com/cosmos/interchain-security/x/ccv/consumer/types"
 	"github.com/cosmos/interchain-security/x/ccv/provider/keeper"
 	"github.com/cosmos/interchain-security/x/ccv/provider/types"
-	providertypes "github.com/cosmos/interchain-security/x/ccv/provider/types"
```

Comment:
Personally, I would have prefer to keep `providertypes` than `types`, as it's more descriptive.

Reply:
I agree with Marius on this one